src/uniprotkb/components/entry/similar-proteins/SimilarProteins.tsx

```diff
@@ -89,14 +87,12 @@ const SimilarProteins: FC<{
     <div id={EntrySection.SimilarProteins}>
       <Card title={nameAndId.name}>
         <Tabs>
-          {Object.values(UniRefEntryType).map(
-            (clusterType) =>
+          {Object.entries(uniRefEntryTypeToPercent).map(
```

Comment:
`representativeId` has been removed from the schema but is referenced at L66, L99-102.  Specifically:
```
        allClusterTypesGroups[key] = groupBy(
          clusterTypeGroups[key],
          'representativeId'
        );
```
Is there a way to get TSC to make sure the string we pass is actually a key?

Reply:
Good point, I think the only possibility is to use a function accessor instead of an accessor string as in this case, this is the only way for TS to detect.
I will review the use of `groupBy` and update accordingly

Reply:
OK, I think this is tackled now

Reply:
Looks good to me